### ISSUE TITLE
Discourse - Lack of visible focus ring around the "Aanmelden" button in the Registreren modal and the "Maak je account" in the Aanmelden modal

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#131]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
<a href="https://community.developer.overheid.nl/t/officiele-bekendmakingen-binnenlandse-zaken-en-koninkrijksrelaties/726">Officiële bekendmakingen (Binnenlandse Zaken en Koninkrijksrelaties)</a>

**Probleembeschrijving:**
Probleem: De "Maak je account"-knop van het Aanmelden-modal en de "Aanmelden"-knop van het Registreren-modal missen een adequate zichtbare focusindicator.

Mogelijke verbeteringen: Voeg een duidelijke focusstatus toe aan de knoppen door middel van een zichtbare focusindicator met voldoende contrast (zoals een stevige omlijning) en zorg ervoor dat de focusstatus te onderscheiden is van andere statussen (hover, active).

**Referentiecode:**
```
<button class="btn-large btn-primary btn btn-text" type="button">
<!----><span class="d-button-label">Maak je account<!----></span>
    </button>
```

**Gerelateerde WCAG:**
WCAG 2.4.7 Focus visible
Level AA

**Referentieafbeelding:**

![The developer tools' console shoes the button Aanmelden is focused but it shows no focus ring around it or any background color changes that would help sighted users see its status](https://github.com/user-attachments/assets/feb261fa-2a8b-46dd-a178-ea27f51c744a)

_**Referentienummer:** A147_
